controllers/front/AttachmentController.php

```diff
@@ -44,7 +45,45 @@ public function postProcess()
         header('Content-Length: '.filesize(_PS_DOWNLOAD_DIR_.$a->file));
         header('Content-Disposition: attachment; filename="'.utf8_decode($a->file_name).'"');
         @set_time_limit(0);
-        readfile(_PS_DOWNLOAD_DIR_.$a->file);
+        //readfile(_PS_DOWNLOAD_DIR_.$a->file);
+        self::readfile_chunked(_PS_DOWNLOAD_DIR_.$a->file);        
         exit;
     }
+
+    /**
+     * http://ca2.php.net/manual/en/function.readfile.php#54295
+     * jjordan@hofmann.es
```

Comment:
we usualy dont put author name on function, you can add yourself in class signature using `@author` PHPDoc